src/redux/reducers/blogReducer.ts

```diff
@@ -6,17 +6,13 @@ const initialState = {
 
 const sortBlogs = (blogs:Iblog[]) => {
   let sortedBlogs = blogs;
-  try {
```

Comment:
@logrdog I believe the purpose of this is for the condition where you have no blogs to sort and then you get an error, so removing this code is not the solution we wanted

Reply:
I thought so also but I wasn't able to assert a test case that I could throw the error. It doesn't reach the catch block even when the object passed into redux is empty, which I am still confused about. I added a test case with an empty object. So I still have to look into that.